docs/reference/cluster/update-desired-nodes.asciidoc

```diff
@@ -58,6 +62,10 @@ this API to let Elasticsearch know about the cluster topology, including future
 changes such as adding or removing nodes. Using this information, the system is
 able to take better decisions.
 
+It's possible to run the update in "dry run" mode by adding the
+`?dry_run` query parameter, or by passing `"dry_run": true` in the request
```

Comment:
> or by passing `"dry_run": true` in the request body

I think this is incorrect, we only support the URI query parameter and I would prefer to keep it that way.

Reply:
Good catch! To be honest, we accept it in both forms.